app/src/main/java/org/wikipedia/analytics/SessionData.kt

```diff
@@ -0,0 +1,60 @@
+package org.wikipedia.analytics
+
+import org.wikipedia.history.HistoryEntry
+import org.wikipedia.util.MathUtil.Averaged
+
+class SessionData {
+    private val leadLatency = Averaged<Long>()
```

Comment:
Please use `MathUtil.Averaged` instead of `Averaged` for overall code style consistency.